src/Microsoft.ML.Data/DataLoadSave/DataOperationsCatalog.cs

```diff
@@ -26,13 +26,15 @@ internal DataOperationsCatalog(IHostEnvironment env)
 
         /// <summary>
         /// Creates a lazy in-memory cache of <paramref name="input"/>.
-        /// Caching happens per-column. A column is only cached when it is first accessed.
-        /// In addition, <paramref name="columnsToPrefetch"/> are considered 'always needed', so all of them
-        /// will be cached whenever any data is requested.
         /// </summary>
+        /// <remarks>
+        /// Caching happens per-column. A column is only cached when it is first accessed.
+        /// In addition, <paramref name="columnsToPrefetch"/> are considered 'always needed', so these columns
+        /// will be cached the first time any data is requested.
+        /// </remarks>
         /// <param name="input">The data view to cache.</param>
```

Comment:
>data view [](start = 36, length = 9)

<see cref="IDataView" />

Reply:
Used the same text as the other filters.